index.html

```diff
@@ -1,8 +1,60 @@
+                </p>
+                </section>
+                <section>
+                <a name="Number 2"></a>
+                <h3>Reactivity</h3>
+                <p>
+                I am what they call a reactive dog. I let my emotions get the best of me. This can come across as a bit intimidating to those who do not know me. I may squeel very loudly at other dogs or lunge towards what interests me (and even run from what terrifies me!). My mom takes extra care to be sure I am in comfortable environments so that I am confident and relaxed. One way she does this is to be as prepared as possible. This is so important when on walks and at the park. Pay attention to your surroundings - what is coming towards or behind, keep an eye out for loose animals, and watch out for any other triggers. Always keep high value treats handy to distract and redirect, and practice situation management. When you are able to, remove your dog from a potential trigger situation while the dog is still calm. Reward and praise your pup every time they have a successful interaction or redirection! And when they don't, because it will happen, redirect with compassion. Remember that reactivity is fueled by the inability to manage emotions, it is not aggression or purposeful disobeying.
+                </p>
+            </section>
+            <section>
+                <a name="#3"></a>
+                <h4>Mental Stimulation</h4>
+                <p>
+                Did you know that you can burn your dog's energy by mental stimulation? Humans are physically unable to fulfill most dog's need for daily exercise. In as few as 10 minutes of training each day, you can make an impact on their energy level. Other great options are nose work games and snuffle mats. These make food time rewarding, as your dog will get to use their foraging and scent skills during the feeding process. I love my snuffle mat!
+                <figure>
+                  <img id="snuffle" src="images/snuffle.jpg" />
+                </figure>
+                </p>
+            </section>
+            <section>
+                <a name= "closing"></a>
+                <p>
+                Dog training can be challenging but also very rewarding. It is an important investment in the quality of life for the entire family. If your dog has excess energy or reactivity concerns, please consider incorporating some of these ideas into your dog's daily routine!
+                </p>
+            </section>
+            <section>
+              <h5>Follow Blaze on Instagram @buggyblaze</h5>
+             </section>
+           </article>
+         </main>
+        <footer>
+          <p>&copy; 2022 Ellen Brady</p>
+        </footer>
    </body>
-</html>
\ No newline at end of file
+</html>
```

Comment:
In the future, you can add some spaces between each section to block together specific, well, sections. It can make it easier to jump around. Otherwise, your formatting is looking good!